nbdev/cli.py

```diff
@@ -210,8 +210,7 @@ def _fetch_from_git(raise_err=False):
         email = run('git config --get user.email').strip()
     except Exception as e:
```

Comment:
@hamelsmu I think we should avoid `except Exception` wherever possible. Instead, constrain to the most explicit exception you can.

Reply:
Ok I fixed this in #723